src/estimation_least_squares/RegressionForm.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import Rheostat from 'rheostat';
 import ReactTooltip from 'react-tooltip';
 import { NumericField } from '../utility_components/NumericField';
+import { InlineMath } from 'react-katex';
```

Comment:
This looks like a great library! I've always used random forks of react-mathjax but have always had minor issues with them.